backends/arm/test/common.py

```diff
@@ -93,7 +93,7 @@ def get_tosa_compile_spec_unbuilt(
 def get_u55_compile_spec(
     macs: int = 128,
     system_config: str = "Ethos_U55_High_End_Embedded",
-    memory_mode: str = "Shared_Sram",
+    memory_mode: str = "Sram_Only",
```

Comment:
what is the rationale for changing the default? I don't see any issues, asking because I am not sure why this was the default before.

cc @freddan80, @zingo